api/src/pcapi/core/offers/factories.py

```diff
@@ -118,6 +118,14 @@ class EventStockFactory(StockFactory):
     bookingLimitDatetime = factory.LazyAttribute(lambda stock: stock.beginningDatetime - datetime.timedelta(minutes=60))
 
 
+class CinemaStockFactory(StockFactory):
```

Comment:
```suggestion
class SyncCinemaStockFactory(StockFactory):
```

C'est bien uniquement les stock synchronisé qui n'ont pas d'heure ?

Reply:
bien vu !

Reply:
j'ai mis `CinemaStockProviderFactory`j'ai l'impression que ça s'appelle comme ça sur l'api